apps/k8s-io/configmap-nginx.yaml

```diff
@@ -362,8 +362,8 @@ data:
         server_name sbom.k8s.io sbom.kubernetes.io
         listen 80;
 
-        rewrite ^/(.*)?/release(\.cert|\.sig|\.sha256|\.sha512)?$ https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-release.spdx$2 redirect;
-        rewrite ^/(.*)?/source(\.cert|\.sig|\.sha256|\.sha512)?$  https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-source.spdx$2 redirect;
+        rewrite ^/(.*)?/release(\.cert|\.sig|\.sha256|\.sha512)?$ https://dl.k8s.io/release/$1/kubernetes-release.spdx$2 redirect;
+        rewrite ^/(.*)?/source(\.cert|\.sig|\.sha256|\.sha512)?$  https://dl.k8s.io/release/$1/kubernetes-source.spdx$2 redirect;
```

Comment:
Is there any drawback for older releases?

Reply:
All the older releases are currently synced to the new bucket.

```bash
http -I -F https://dl.k8s.io/release/v1.22.0/kubernetes-release.spdx.sha256
HTTP/1.1 200 OK
Accept-Ranges: bytes
Age: 0
Cache-Control: private, no-store
Connection: keep-alive
Content-Length: 64
Content-Type: application/octet-stream
Date: Wed, 11 Sep 2024 09:48:37 GMT
ETag: "f98d02098f161ce5c02040f58cf3f4d2"
Last-Modified: Fri, 17 Nov 2023 20:16:08 GMT
access-control-allow-origin: *

47eb5acc2319c773ceab650b0278348bc9411bc9fc40d8d63da4b2508ac119ba
```

We shouldn't have any drawback at least for those with provenance and sboms (k8s >= 1.22.0 IIRC)